From 38c891014bafefa6c56fff335e951e7e931320d1 Mon Sep 17 00:00:00 2001
From: graysky <graysky@archlinux.us>
Date: Tue, 22 Dec 2020 14:55:32 -0500
Subject: [PATCH 5/5] update for py3 and kodi 19 compatibility

also chmod -x several files that do not need this flag
---
 addon.xml              | 2 +-
 resources/entries.json | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 mode change 100755 => 100644 addon.xml
 mode change 100755 => 100644 resources/entries.json

diff --git a/addon.xml b/addon.xml
old mode 100755
new mode 100644
index c9c4b0d..e72f4b1
--- a/addon.xml
+++ b/addon.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="screensaver.atv4" name="Aerial" version="1.5.1" provider-name="enen92">
 	<requires>
-		<import addon="xbmc.python" version="2.25.0"/>
+		<import addon="xbmc.python" version="3.0.0"/>
 	</requires>
 	<extension point="xbmc.ui.screensaver" library="entrypointscreensaver.py" />
 	<extension point="xbmc.python.script" library="entrypointscript.py">
diff --git a/resources/entries.json b/resources/entries.json
old mode 100755
new mode 100644
-- 
2.29.2

